app/src/main/java/com/sun/mangareader01/ui/search/SearchPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package com.sun.mangareader01.ui.search
+
+import com.sun.mangareader01.data.model.MangaSearchResponse
+import com.sun.mangareader01.data.source.local.OnLoadedDataCallback
+import com.sun.mangareader01.data.source.repository.MangaRepository
+
+class SearchPresenter(
+    private val searchView: SearchFragment,
+    private val mangaRepository: MangaRepository
```

Comment:
```suggestion
    private val repository: MangaRepository
```
nếu trong class chỉ có 1 repository e có thể đặt tên ntn cho dễ hiểu